core/src/main/java/org/stellar/anchor/paymentservice/PaymentService.java

```diff
@@ -9,20 +9,13 @@
  * Contains the interface methods for a payment service. It can be implemented in different
  * networks, like Stellar, Circle, Wyre or others.
  */
-@SuppressWarnings("unused")
```

Comment:
Nice! This should be avoided when possible although sometimes, it is not possible if we are building a library)

Reply:
One nice side-effect of using a mono repo is that we can remove when they are used by other subprojects 👍  resolving.